core/tests/protractor/subscriptions.js

```diff
@@ -86,10 +82,10 @@ describe('Subscriptions functionality', function() {
     creatorDashboardPage.navigateToSubscriptionDashboard();
     expect(element.all(by.css(
```

Comment:
With your new preferences page object, this shouldn't be referencing elements of the DOM directly.

Reply:
Sorry, What's DOM?

Reply:
https://en.wikipedia.org/wiki/Document_Object_Model

Basically, the HTML tags.

Reply:
As per my understanding, it has to deal with the creatordashboard page and not the preferences page. So, I think it should be left untouched as far as this PR is concerned.

Reply:
Ah yes, you're right. Thanks for catching that!